### DISCUSSION TITLE
How to reverify my academic affiliation, If I’m still eligible.

### DISCUSSION THREAD

**Discussion by sandeepvinnu** [community/community#134055] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,

I read an article on GitHub [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student#expiration-and-renewals), but it needs updating. Could anyone guide me on the new process to reverify?

**Comment by davevad93** [comment, marked as answer]
Hi @sandeepvinnu ,

The student validation lasts for 2 years, after which you will need to revalidate your student status by re-applying on the [GitHub Education application page](https://education.github.com/discount_requests/application) and submitting the required documentation as you did the first time.
